packages/arco-lib/src/components/Link.tsx

```diff
@@ -11,12 +11,20 @@ const LinkStateSpec = Type.Object({});
 const exampleProperties: Static<typeof LinkPropsSpec> = {
   disabled: false,
   hoverable: true,
-  status: 'warning',
+  status: 'default',
   href: 'https://www.smartx.com/',
   content: 'Link',
 };
 
-const options = {
+
+const statusMap = {
+  default: undefined,
+  success: 'success',
+  error: 'error',
+  warning: 'warning',
+} as const;
```

Comment:
why as const?

Reply:
This is because Link's status property accepts the types `"success" | "error" | "warning" | undefined` Therefore the `as const ` keyword is needed to avoid literal types being widened(e.g. no going from "success" to string)